widget/ui/src/components/TextField/TextField.types.ts

```diff
@@ -0,0 +1,15 @@
+export type PropTypes = {
+  label?: string;
+  prefix?: React.ReactNode;
+  suffix?: React.ReactNode;
+  size?: 'small' | 'large';
+  variant?: 'contained' | 'outlined' | 'ghost';
+  fullWidth?: boolean;
+  type?: 'dark' | 'light';
```

Comment:
The type property overrides the input type attribute . Please rename or remove this property.